Human-rights

Andriykooo opened the pull request:
- [DEMO LINK](https://Andriykooo.github.io/layout_human-rights/) <br>
- [TEST REPORT LINK](https://Andriykooo.github.io/layout_human-rights/report/html_report/)

Andriykooo committed:
Human-rights

varseniuk commented on src/index.html:
```diff
@@ -1,14 +1,104 @@
+        Whereas recognition of the inherent dignity and of the equal and inalienable rights of all members of the human family is the foundation of freedom, justice and peace in the world,<br> Whereas disregard and contempt for human rights have resulted
+        in barbarous acts which have outraged the conscience of mankind, and the advent of a world in which human beings shall enjoy freedom of speech and belief and freedom from fear and want has been proclaimed as the highest aspiration of the common
+        people,
+      <br> Whereas it is essential, if man is not to be compelled to have recourse, as a last resort, to rebellion against tyranny and oppression, that human rights should be protected by the rule of law,<br> Whereas it is essential to promote the development of friendly relations between nations,
```
```suggestion
      <br>
      Whereas it is essential, if man is not to be compelled to have recourse, as a last resort, to rebellion against tyranny and oppression, that human rights should be protected by the rule of law,
      <br>
      Whereas it is essential to promote the development of friendly relations between nations,
```

varseniuk commented on src/index.html:
```diff
@@ -1,14 +1,104 @@
+      <li>The family is the natural and fundamental group unit of society and is entitled to protection by society and the State.</li>
+    </ol>
+    <h2>
+      <a href="https://www.ohchr.org/EN/UDHR/Documents/UDHR_Translations/eng.pdf">Articles 17 to 30</a>
```
```suggestion
      <a href="https://www.ohchr.org/EN/UDHR/Documents/UDHR_Translations/eng.pdf">
        Articles 17 to 30
      </a>
```
It's hard to notice the content of the tag with your formatting

varseniuk commented on src/index.html:
```diff
@@ -1,14 +1,104 @@
+    </h2>
+  </body>
+
+</html>
```
add an empty line at the end

varseniuk commented on src/style.css:
```diff
@@ -0,0 +1,20 @@
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
```
As task description says
`Remove all the outer spacing for all the heading and paragraphs.`
Let's add rule for `p` as well

Andriykooo committed:
Human-rights

-deleted spaces in tag <p>
-formatted text after tag <br>
-formatted tag <a>
-removed all outer spacing in tag <p>
+epmty line at the end

varseniuk commented on src/style.css:
```diff
@@ -0,0 +1,24 @@
+  color: green;
+}
+
+p {
+  margin: 0;
+}
```
```suggestion
h1,
p {
  margin: 0;
}
```
Don't repeat yourself. You have `margin: 0` both in your `h1` and `p` selector. You can write it like I've shown above and remove the rule `margin: 0` from `h1` selector

varseniuk reviewed (changes requested):
Let's make one small improvement in css

Andriykooo committed:
Human-rights

- CSS fix (dont repeat yourself, i got it ;)

varseniuk reviewed (approved):
👍
